### ISSUE TITLE
feature: Digitally singed application

### ISSUE THREAD

**Issue by ghost** [webcatalog/webcatalog-legacy#1029]
# Feature Request
**Specifications:**
- App Version: 24.7.0
- OS (macOS, Windows, Linux): Windows

**Description:**
I prefer to only install digitally signed applications on my home laptop. On my work laptop I can only install digitally signed applications (without administrator privileges). Would be nice to have the Webcatalog installer digitally signed such that I can install it on my work laptop as well.

**Comment by quanglam2807** [comment]
I'm working on this. It'll be signed in a few days.

Also, you don't need to recreate the issues. WebCatalog & Singlebox are developed in parallel. So new features will be added to both.

**Comment by ghost** [comment]
Perfect.
_Apologies for the duplication_

**Comment by quanglam2807** [comment]
I'm sorry but this would be delayed by 1-2 weeks.

**Comment by quanglam2807** [comment]
Resolved with https://github.com/webcatalog/webcatalog-app/pull/1095! It took much longer than I hoped.
